rpm/libiphb.spec

```diff
@@ -84,8 +61,6 @@ rm -rf %{buildroot}
 %defattr(-,root,root,-)
 %{_libdir}/libiphb.so.*
 %doc COPYING AUTHORS INSTALL README NEWS ChangeLog
```

Comment:
IMHO let's just keep them in instead of doing qa-stage-dependend conditionals here.